### PR TITLE
[MVC-988] Website Responsiveness: Text adaptation to resolution

### PR DESCRIPTION
**Pull request by nashthecoder**
Added margin-top and margin-bottom on text so that the Title doesnt move up to the photo as in the example given.  
See Video:
https://user-images.githubusercontent.com/15178823/117426396-859cce00-af2c-11eb-9183-f386e9b3ccd7.mov
The fix is currently cosmetic because in working on this I discovered that:

- wrapper divs have been used inside containers to wrap around more than one element. Convention dictates that container are used to contain more than one element and wrappers to wrap a single object to provide more functionality and interface to it.  
- the div class names are very confusing , for some elements it would help to give them ids just to make the code clearer and cleaner. I see this best tackled during the sprint on the architectural tech debt.

### CODE REVIEW

**Comment by tiagovarela** [comment]
Hey @nashthecoder this PR includes the `privacy_policy.html` change from the `programatic-privacy-policy` branch which I think is not related to this PR and should be removed.

**Comment by nashthecoder** [comment]
On it. Let me remove it.

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +0,0 @@
-{
-    "terminal.integrated.fontFamily": "Hack Nerd Font"
-}
```
Comment (raguiar9080) [inline .vscode/settings.json]: I think this is not related to the actual codebase, this might be a case where we want to have this file in .gitignore

File: _layouts/privacy_policy.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    </div>
+
+    <div class="success-story-wrapper">
+{% for policy in page.Policies %}
+<div data-lang="{{ policy.Language | slugify: "latin" }}">
+        {{ policy.Text | markdownify }}
+</div>
+{% endfor %}
+    </div>
+</div>
```
Comment (raguiar9080) [inline _layouts/privacy_policy.html:29]: all around indentation here is off. I think it makes thinks a lot clearer if any open html element adds indentation and closing removes one.

File: _layouts/privacy_policy.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+if (navLanguages[i].dataset.navLang === lang) {
+        navLanguages[i].style = "color: #1D214F;";
+} else {
+        navLanguages[i].style = "color: #999;";
+}
+    }
+} else {
+    window.location.hash = "english";
+    display_lang("english");
+}
```
Comment (raguiar9080) [inline _layouts/privacy_policy.html:56]: Same here in the JS and the method below

**Review by raguiar9080 — COMMENTED** [review]
As Varela stated, some changes on the `privacy_policy.html` that don't relate to the User story. Also, not very critical but the first video doesn't really show anything, it's only a 2 sec one. The other video is quite good and easy to see, nice job on that 👍

File: .vscode/settings.json (hunk 1 above)
Comment (nashthecoder) [reply .vscode/settings.json]: Working on it.

**Comment by nashthecoder** [comment]
@raguiar9080  This is the correct branch https://github.com/Triggerise/triggerise.github.io/tree/text-reponsive-fix 

There was a mixup with the code initially pushed caused by changing laptops and not pulling and not confirming which branch I was on, while on the new machine. This I have since sorted and removed the previously added "pragmatic-polic-change code."
